### DISCUSSION TITLE
which version is single-precision computing

### DISCUSSION THREAD

**Discussion by university12138** [brucefan1983/GPUMD#913] [General]
Hello developer, I want to ask which version is single-precision computing, thank you！

**Comment by brucefan1983** [comment]
Could you give more details to your question?

**Comment by university12138** [comment]
Because when I calculate the thermal conductivity of silicon with a length of 2000 nm, the potential function is sw. Because my calculation time is relatively long, the cluster has a limit on the calculation time, resulting in the calculation of 2000 nm cannot be completed, so I want to use the gpumd version of single-precision calculation to solve the calculation time problem.

**Comment by brucefan1983** [comment]
Perhaps there is no released version with pure single precision in MD part. I think you need to modify code to achieve it.

**Comment by university12138** [comment]
thanks.
